### ISSUE TITLE
Timeout argument on 'New Browser'

### ISSUE THREAD

**Issue by vinismarques** [MarketSquare/robotframework-browser#2064]
**Describe the bug**
The timeout argument of the `New Browser` keyword seems to be broken.

**To Reproduce**
1. Execute this line of code:
    ```
    New Browser    browser=chromium    headless=False    timeout=0s
    ```
4. Error is thrown
5. Variants like `timeout=0` do not work as well

**Expected behavior**
Error is thrown:

```
Error: browserType.launch: timeout: expected number, got string [[-]](vscode-webview://1v30vc43hfdiemc74gulcl3244gg5isefsrn8nq2vlc7lvvmpsv6/index.html?id=11fd3aa1-5851-4930-9eb6-d1c68c79cbb2&swVersion=4&extensionId=robocorp.robotframework-lsp&platform=electron&vscode-resource-base-authority=vscode-resource.vscode-cdn.net&parentOrigin=vscode-file%3A%2F%2Fvscode-app#)
Traceback (most recent call last):
  File "c:\Users\vinic\.vscode\extensions\robocorp.robotframework-lsp-0.47.2\src\robotframework_ls\vendored\robotframework_interactive\interpreter.py", line 375, in evaluate
    return self._evaluate(code)
  File "c:\Users\vinic\.vscode\extensions\robocorp.robotframework-lsp-0.47.2\src\robotframework_ls\vendored\robotframework_interactive\interpreter.py", line 482, in _evaluate
    facade.run_test_body(context, test)
  File "c:\Users\vinic\.vscode\extensions\robocorp.robotframework-lsp-0.47.2\src\robotframework_ls\vendored\robotframework_interactive\robotfacade.py", line 67, in run_test_body
    BodyRunner(context, templated=False).run(test.body)
  File "C:\Users\vinic\AppData\Local\robocorp\holotree\b1f3c244e_6cd60f16\lib\site-packages\robot\running\bodyrunner.py", line 61, in run
    raise ExecutionFailures(errors)
robot.errors.ExecutionFailures: Error: browserType.launch: timeout: expected number, got string
```

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chromium
 - robotframework-browser==13.0.0
 - rpaframework==14.2.0

**Comment by aaltat** [comment]
This seems to happen outside of the Browser library code, because Browser library relies on the Robot Framework to perform argument conversion. Which version of Robot Framework you are using and if you are not using the latest could you try with the latest Robot Framework version?

**Comment by vinismarques** [comment]
I am using Robot Framework 5.0.1, so I guess it is the latest already. Were you able to execute step 1 in the "To Reproduce" topic without an issue?

**Comment by aaltat** [comment]
@allcontributors please add @vinismarques for bugs.

**Comment by luchoagomezt** [comment]
I can reproduce the issue only when the timeout is 0:

**Comment by luchoagomezt** [comment]
[PR 2108](https://github.com/MarketSquare/robotframework-browser/pull/2108) fixes this issue.

**Comment by aaltat** [comment]
Fixed in #2108
